### ISSUE TITLE
Projects windows gets lost after selecting folder

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#3257]
### Short Summary
When creating a new project. 
After selecting the folder, to store the project file. The "new project" screen stays hidden behind the main Komodo Window. You have to focus the window manually.

### Steps to Reproduce
* Create a new project
* Select a other folder location

You will end up with the projects screen hidden.

### Expected results

The "new project" screen gets focused in.

### Actual results

The "new project" screen stays hidden.

### Platform Information

Komodo IDE, version 11.0.0, build 90668, platform win32-x86

**Comment by mitchell-as** [comment]
@cgchoffman are you able to reproduce this on Windows? I am not able to reproduce this on Linux.

**Comment by cgchoffman** [comment]
Yes i'm able to reproduce this on Windows.  In fact I'm certain I already filed this issue. Looking...
